code/modules/client/preferences/body_size.dm

```diff
@@ -0,0 +1,16 @@
+/datum/preference/choiced/body_size
+	category = PREFERENCE_CATEGORY_SECONDARY_FEATURES
+	savefile_identifier = PREFERENCE_CHARACTER
+	priority = PREFERENCE_PRIORITY_BODY_TYPE
+	savefile_key = "body_size"
+	savefile_identifier = PREFERENCE_CHARACTER
+
+/datum/preference/choiced/body_size/init_possible_values()
+	return list("Normal", "Short", "Tall")
```

Comment:
(OPTIONAL)
Do you think it'd be worth changing the order of these to 'Short', 'Normal', 'Tall'?
Neither of the servers it's ported from have it this way, but to me at least it seems a little bit more natural.
![NVIDIA_Share_d1GUrv0ZmF](https://user-images.githubusercontent.com/57483089/166242832-153fe6c6-0523-4f43-844d-f55a938fc848.gif)
<hr>

```suggestion
/datum/preference/choiced/body_size/init_possible_values()
	return list("Short", "Normal", "Tall")
```